Frontend/ui-customer/src/Screens/CustomerUtils/TripDetailsScreen/View.purs

```diff
@@ -166,40 +167,66 @@ lostAndFoundView push state =
 ---------------------- tripIdView ---------------------------
 tripIdView :: forall w . (Action -> Effect Unit) -> ST.TripDetailsScreenState -> PrestoDOM (Effect Unit) w
 tripIdView push state =
-  linearLayout
-  [ orientation VERTICAL
+  let rideType = case state.data.vehicleVariant of
+                    Just variant -> getVariantRideType (show variant)
+                    Nothing      -> getString AC_CAB
+  in
+  frameLayout
```

Comment:
can you please explain where can we use map ?

Reply:
inside the frame layout we have two layout one for trip id another for ride type and both are similar only . So we can use map for them.

Reply:
> instead of frameLayout can we use linear layout here

linearLayout will not be helpful as we want next layout to be started exactly after half of screen width

Reply:
will frameLayout create any issue ?

Reply:
frameLayout consumes more resources to render compared to relative layout

Reply:
> linearLayout will not be helpful as we want next layout to be started exactly after half of screen width

in that case you can use weight that would be equally divided